event-sourcing/event-source-api/src/main/java/uk/gov/justice/services/eventsourcing/source/api/service/core/EventEntry.java

```diff
@@ -41,8 +41,8 @@ public String getName() {
         return name;
     }
 
-    public long getSequenceId() {
-        return sequenceId;
+    public long getVersion() {
```

Comment:
is version the correct name?

It doesn't seem to be....

Reply:
Yes for Events it is version and sequenceNumber is relevant for EventStreamObject